v0/destinations/slack/transform.js

```diff
@@ -305,6 +307,12 @@ function process(event) {
   const respList = [];
   let response;
   const { message, destination } = event;
+  if (!message.type) {
```

Comment:
[Non-blocking comment] Can we check if events without type get/should be filtered at the gateway/server level?
@akashrpo

Reply:
It will throw an error as - Cannot read property 'toLowerCase' of undefined